channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -268,6 +271,13 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControls {
                     let ctx = createProcessTelemetryContext(null, config, core);
                     // getExtCfg only finds undefined values from core
                     let senderConfig = ctx.getExtCfg(identifier, defaultAppInsightsChannelConfig);
+                    if (senderConfig.corsPolicy){
+                        if (senderConfig.corsPolicy === "same-origin" || senderConfig.corsPolicy === "same-site" || senderConfig.corsPolicy === "cross-origin") {
```

Comment:
nice check.

Lets use a local variable for all of the `senderConfig.corsPolicy` usages as while the `senderConfig` will get minified all of the `.corsPolicy` references won't which has 2 side effects.
1) The minified size 
2) Performance, it's not obvious but every time you reference a property of an object the runtime needs to "lookup" the value so you end up spending a few cycles running code unnecessarily, it's not a "big" problem here as this code only runs when there is a config change, but every little bit adds up.
- Also as this is a dynamic config reference, each lookup will actually run all of the dynamic config "tracking" code (a couple of function calls) for each reference.